### PR TITLE
Orca tf lenet mnist slim & keras example

### PR DESCRIPTION
**Pull request by cyita**

### CODE REVIEW

File: pyzoo/zoo/examples/orca/learn/tf/lenet_mnist_graph.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+
+def accuracy(logits, labels):
+    predictions = tf.argmax(logits, axis=1, output_type=labels.dtype)
+    is_correct = tf.cast(tf.equal(predictions, labels), dtype=tf.float32)
+    return tf.reduce_mean(is_correct)
+
+
+def main(max_epoch):
+    sc = init_nncontext()
```
Comment (hkvision) [inline pyzoo/zoo/examples/orca/learn/tf/lenet_mnist_graph.py:37]: use init_orca_context

File: pyzoo/zoo/examples/orca/learn/tf/lenet_mnist_graph.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+    # get DataSet
+    mnist_train = tfds.load(name="mnist", split="train")
+    mnist_test = tfds.load(name="mnist", split="test")
+
+    images = tf.placeholder(dtype=tf.float32, shape=(None, 28, 28, 1))
```
Comment (qiyuangong) [inline pyzoo/zoo/examples/orca/learn/tf/lenet_mnist_graph.py:43]: Add comment about input.

File: pyzoo/zoo/examples/orca/learn/tf/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+## Run tf graph model example after pip install
+
+```bash
+export MASTER=local[4]
+export SPARK_DRIVER_MEMORY=2g
+python graph_lenet.py
+```
+## Run tf graph model example with prebuilt package
+
+```bash
```
Comment (qiyuangong) [inline pyzoo/zoo/examples/orca/learn/tf/README.md:34]: Maybe we should add this example into auto-test. :)

**Comment by cyita** [comment]
Jenkins passed http://10.239.47.210:18888/job/ZOO-PR-Validation/4112/

File: pyzoo/zoo/examples/orca/learn/tf/lenet_mnist_keras.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    mnist_train = mnist_train.map(normalize_img, num_parallel_calls=tf.data.experimental.AUTOTUNE)
+    mnist_test = mnist_test.map(normalize_img, num_parallel_calls=tf.data.experimental.AUTOTUNE)
+
+    model = tf.keras.Sequential(
+        [tf.keras.layers.Flatten(input_shape=(28, 28, 1)),
+         tf.keras.layers.Dense(64, activation='relu'),
+         tf.keras.layers.Dense(64, activation='relu'),
+         tf.keras.layers.Dense(10, activation='softmax'),
+         ]
+    )
```
Comment (jason-dai) [inline pyzoo/zoo/examples/orca/learn/tf/lenet_mnist_keras.py:44]: better to construct a lenet

File: pyzoo/zoo/examples/orca/learn/tf/lenet_mnist_keras.py [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+    model.compile(optimizer=tf.keras.optimizers.RMSprop(),
```
Comment (jason-dai) [inline pyzoo/zoo/examples/orca/learn/tf/lenet_mnist_keras.py:46]: shall we move `compile` to `Estimator.from_keras`, so that is is consistent with tf2 @yangw1234 ?

**Comment by cyita** [comment]
jenkins passed http://10.239.47.210:18888/view/ZOO-PR-PIPELINE/job/ZOO-PR-Validation/4121/

File: pyzoo/zoo/examples/orca/learn/tf/lenet_mnist_keras.py [hunk 6]
```diff
@@ -17,6 +17,7 @@
 
 import tensorflow as tf
 import tensorflow_datasets as tfds
+import numpy as np
```
Comment (jason-dai) [inline pyzoo/zoo/examples/orca/learn/tf/lenet_mnist_keras.py:20]: where is `np` used?
